src/main/java/org/sciborgs1155/robot/Drive/DriveIO.java

```diff
@@ -0,0 +1,19 @@
+package org.sciborgs1155.robot.Drive;
+
+import monologue.Logged;
+
+public interface DriveIO extends Logged {
```

Comment:
i think with the changes i proposed, this would end up with setVoltages, the getSpeeds for left and right (but not the speed()), updateState(), and getPose()

Reply:
ko